src/components/LessonLink/LessonLink.test.jsx

```diff
@@ -0,0 +1,38 @@
+import LessonLink from "./LessonLink";
+import { customRender } from "../../utils/testUtils";
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+
+describe("LessonLink component", () => {
+  it("matches the snapshot taken", () => {
+    const { container } = customRender(<LessonLink />);
+    expect(container).toMatchSnapshot();
+  });
+
+  it("matches the snapshot taken with props", () => {
+    const { container } = customRender(
+      <LessonLink courseDuration={"10 min"} courseTitle={"blablabla"} />
+    );
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
Can you change this to just one snapshot using the props, dont need both :D

Reply:
deleted one prop.

Reply:
50% of all snapshots snapped out of existence